src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -77,9 +77,7 @@ const StudentApplication = ({ userProfile, authentication }) => {
    * Loads the user's saved apartment application, if one exists
    */
   useEffect(() => {
-    const handleError = (_error) => {
-      // No existing application was found in the database,
-      // or an error occurred while attempting to load the application
+    const initializeNewApplication = () => {
       setApplicationID(null);
       setEditorUsername((prevEditorUsername) => prevEditorUsername ?? userProfile.AD_Username);
```

Comment:
There should probably also be a `setApplicationDetails`, since that contains all the other variables describing the state of the application

Reply:
I'm guessing the reason you didn't include that originally was because `applicationDetails` has a default value? But you're right, I think this is the cleaner way to do it.